Header  header
float64 speed    # ratio to the maximum speed   (-1.0 <= speed   <= 1.0)
float64 current  # ratio to the maximum current (-1.0 <= current <= 1.0)
